### PR TITLE
sops 1.10 (new formula)

### PR DESCRIPTION
**Pull request by robotblake**
### All Submissions:

- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew/pulls) for the same update/change?

### New Formulae Submissions:

- [x] Does your submission pass
`brew audit --strict --online <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Have you built your formula locally prior to submission with `brew install <formula>`?

### CODE REVIEW

File: Library/Formula/sops.rb [hunk 1]
```diff
@@ -0,0 +1,120 @@
+class Sops < Formula
+  desc "Secrets OPerationS (sops) is an editor of encrypted files"
```
Comment (dunn) [inline Library/Formula/sops.rb]: You don't need to include the name in the description.  Just something like "Encrypted file editor" is better.

File: Library/Formula/sops.rb [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+  end
+
+  resource "six" do
+    url "https://pypi.python.org/packages/source/s/six/six-1.10.0.tar.gz"
+    sha256 "105f8d68616f8248e24bf0e9372ef04d3cc10104f1980f54d57b2ce73a5ad56a"
+  end
+
+  def install
+    ENV.prepend_create_path "PYTHONPATH", libexec/"vendor/lib/python2.7/site-packages"
+    %w[boto3 botocore cffi cryptography docutils enum34 futures idna ipaddress jmespath pyasn1 pycparser python-dateutil ruamel.base ruamel.ordereddict ruamel.yaml setuptools six].each do |r|
```
Comment (dunn) [inline Library/Formula/sops.rb]: Unless you're only operating on some of the resources here, you can do 
```ruby
resources.each do |r|
...
```
That will make it easier in the future to add more dependencies if necessary.

File: Library/Formula/sops.rb [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+
+    ENV.prepend_create_path "PYTHONPATH", libexec/"lib/python2.7/site-packages"
+    system "python", *Language::Python.setup_install_args(libexec)
+
+    bin.install Dir[libexec/"bin/*"]
+    bin.env_script_all_files(libexec/"bin", :PYTHONPATH => ENV["PYTHONPATH"])
+  end
+
+  test do
+    system "#{bin}/sops", "-v"
```
Comment (dunn) [inline Library/Formula/sops.rb]: Any chance of a better test? Something that checks more of the functionality would be great.

**Comment by dunn** [comment]
The test is failing:
```
==> /usr/local/Cellar/sops/1.10/bin/sops -v
Traceback (most recent call last):
  File "/usr/local/Cellar/sops/1.10/libexec/bin/sops", line 9, in <module>
    load_entry_point('sops==1.10', 'console_scripts', 'sops')()
  File "/usr/local/Cellar/sops/1.10/libexec/vendor/lib/python2.7/site-packages/pkg_resources/__init__.py", line 542, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/usr/local/Cellar/sops/1.10/libexec/vendor/lib/python2.7/site-packages/pkg_resources/__init__.py", line 2542, in load_entry_point
    return ep.load()
  File "/usr/local/Cellar/sops/1.10/libexec/vendor/lib/python2.7/site-packages/pkg_resources/__init__.py", line 2202, in load
    return self.resolve()
  File "/usr/local/Cellar/sops/1.10/libexec/vendor/lib/python2.7/site-packages/pkg_resources/__init__.py", line 2208, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/usr/local/Cellar/sops/1.10/libexec/lib/python2.7/site-packages/sops/__init__.py", line 26, in <module>
    import ruamel.yaml
ImportError: No module named ruamel.yaml

File: Library/Formula/sops.rb (hunk 3 above)
Comment (robotblake) [inline Library/Formula/sops.rb]: I'm not sure there's an easy way to test the functionality without depending on gpg / AWS and messing with the users keychain / KMS stuff.

**Comment by robotblake** [comment]
Ugh, hadn't pushed my updated commit.  Made the description and resources changes and should pass the tests now.

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Pull request closed without merging**